.gitignore

```diff
@@ -0,0 +1 @@
+.vscode
```

Comment:
No local .gitignores please. 

Better setup your global .gitignore so it is avoided everywhere. Please remove this